### PR TITLE
docs(UPM-2380): Added docs for using shell script to get API tokens

### PR DESCRIPTION
**Pull request by skrivov-edb**
## What Changed?

Added docs for using shell script to get API tokens

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [x] This PR adds new content
- [ ] This PR changes existing content
- [ ] This PR removes existing content
- [ ] This PR is for a release, please add this tag:

### CODE REVIEW

**Review by xugy99 — APPROVED** [review]

File: product_docs/docs/edbcloud/beta/reference/index.mdx [hunk 1]
```diff
@@ -8,11 +8,13 @@ The API reference documentation is available from the [EDB Cloud portal](https:/
...
 To access the API, you need a token. The high-level steps to obtain a token are:
 1. [Query the authentication endpoint](#query-the-authentication-endpoint).
-2. [Request the device code](#request-the-device-code).
+2. [Request the device code](#request-the-device-code-using-curl).
 3. [Authorize as a user](#authorize-as-a-user).
-4. [Request the token](#request-the-token).
+4. [Request the token](#request-the-token-using-curl).
 
 
+You can perform all the steps using CURL, but EDB provides an optional script to simplify getting and refreshing your tokens. The script automates steps 2-4. See [Using the get-token Script](#using-the-get-token-script) for details.
```
Comment (ba-edb) [inline product_docs/docs/edbcloud/beta/reference/index.mdx:16]:
```suggestion
You can perform all the steps using `curl`, but EDB provides an optional script to simplify retrieving and refreshing your tokens. The script automates steps 2-4. See [Using the get-token Script](#using-the-get-token-script) for details.
```

File: product_docs/docs/edbcloud/beta/reference/index.mdx [hunk 2]
```diff
@@ -42,7 +44,7 @@ AUDIENCE="https://portal.edbcloud.com/api"
 ```
 The following example calls show use these environment variables.
 
-## Request the Device code
+## Request the Device Code Using Curl
```
Comment (ba-edb) [inline product_docs/docs/edbcloud/beta/reference/index.mdx:47]:
```suggestion
## Request the Device Code Using `curl`
```

File: product_docs/docs/edbcloud/beta/reference/index.mdx [hunk 3]
```diff
@@ -92,7 +96,7 @@ DEVICE_CODE=KEOY2_5YjuVsRuIrrR-aq5gs
 6. Log in with your Azure AD credentials.
 
 
-## Request the Token
+## Request the Token Using CURL
```
Comment (ba-edb) [inline product_docs/docs/edbcloud/beta/reference/index.mdx:99]:
```suggestion
## Request the Token Using `curl`
```

File: product_docs/docs/edbcloud/beta/reference/index.mdx [hunk 4]
```diff
@@ -171,11 +175,12 @@ Example response:
 ```
 
 
-## Refreshing your Token
+## Refresh your Token 
 
 You can use the refresh token to get a new access token. Usually you need a new access token only after the previous one expires or when gaining access to a new resource for the first time. You shouldn't call the endpoint to get a new access token every time you call an API. There are rate limits that throttle the amount of requests to the endpoint that can be executed using the same token from the same IP.
 
-To refresh your token, make a POST request to the `/oauth/token` endpoint in the Authentication API, using `grant_type=refresh_token`. For example:
+### Refresh your Token Using CURL
```
Comment (ba-edb) [inline product_docs/docs/edbcloud/beta/reference/index.mdx:182]:
```suggestion
### Refresh your Token Using `curl`
```

File: product_docs/docs/edbcloud/beta/reference/index.mdx [hunk 5]
```diff
@@ -171,11 +175,12 @@ Example response:
 ```
 
 
-## Refreshing your Token
+## Refresh your Token
```
Comment (ba-edb) [inline product_docs/docs/edbcloud/beta/reference/index.mdx:178]:
```suggestion
## Refresh your Token
```

File: product_docs/docs/edbcloud/beta/reference/index.mdx [hunk 6]
```diff
@@ -205,10 +210,63 @@ The response of this API call includes the `access_token`. For example:
 
 
 
+## Using the get-token Script
```
Comment (ba-edb) [inline product_docs/docs/edbcloud/beta/reference/index.mdx:213]: Should we put "get-token" in `monospace` everywhere in this document? It reads a bit strangely to me as plain text, but I don't know the prevailing style well.

Comment (ba-edb) [inline product_docs/docs/edbcloud/beta/reference/index.mdx:213]: Also, this should be called something like "Request Your Token Using `get-token.sh`, right? To mirror the other heads? Same on L236 below.

**Review by skrivov-edb — COMMENTED** [review]
LGTM

**Pull request merged**